seed/tests/test_utils_geocode.py

```diff
@@ -0,0 +1,207 @@
+# !/usr/bin/env python
+# encoding: utf-8
+"""
+Test Geocoding of Properties and Tax Lots
+
+On first run, HTTP request/responses are truely sent and received.
+On subsequent runs on the same machine, API request/responses are
+intercepted/mocked by VCR. To execute an actual HTTP request/response
+(and not use mocked data), delete the vcr_cassette files.
+"""
+
+import vcr
+
+from django.contrib.gis.geos import Point
+
+from django.test import TestCase
+
+from seed.landing.models import SEEDUser as User
+
+from seed.models.properties import PropertyState
+from seed.models.tax_lots import TaxLotState
+
+from seed.test_helpers.fake import (
+    FakePropertyStateFactory,
+    FakeTaxLotStateFactory,
+)
+
+from seed.utils.geocode import geocode_addresses
+from seed.utils.geocode import long_lat_wkt
+from seed.utils.organizations import create_organization
+
+
+def batch_request_uri_length_matcher(r1, r2):
```

Comment:
In the case of multiple locations being batched into one API/HTTP request, it was possible for the locations to be listed in a different order in between test runs. Because of that, VCR would consider this different order as a different request, and throw errors.

Rather than changing the method to preserve order, this method was created to only compare the length of the URI when considering if the API/HTTP request is the same as a previous run's.